### PR TITLE
[DEVX-609]: Model Export Failures Bug Resolution

### PR DESCRIPTION
**Pull request by srikanthbachala20**
### Why

* Authentions headers missing, so unable to download the file

### How

* Added authentication headers

### CODE REVIEW

**Review by sanjaychelliah — COMMENTED** [review]
LGTM

File: clarifai/client/model.py [hunk 1]
```diff
@@ -1295,15 +1296,22 @@ def _download_exported_model(
...
+      response = session.get(model_export_url, stream=True)
       response.raise_for_status()
 
       with open(local_filepath, 'wb') as f:
         progress = tqdm(
             total=model_export_file_size, unit='B', unit_scale=True, desc="Exporting model")
         for chunk in response.iter_content(chunk_size=8192):
-          f.write(chunk)
-          progress.update(len(chunk))
+          if chunk:
```
Comment (nitinbhojwani) [inline clarifai/client/model.py:1310]: what is the intention behind this if else block?
We are not expecting chunk to be empty or None ever. We can undo this change.

**Review by nitinbhojwani — CHANGES_REQUESTED** [review]

File: clarifai/client/model.py (hunk 1 above)
Comment (srikanthbachala20) [reply clarifai/client/model.py:1310]: Intially, i thought there is some issue with the chucks, so I have added this if else block.I will undo this change now.

Comment (srikanthbachala20) [reply clarifai/client/model.py]: Done

**Review by sanjaychelliah — APPROVED** [review]

**Review by nitinbhojwani — APPROVED** [review]
